repository.yaml

```diff
@@ -0,0 +1,3 @@
+templates:
+  - name: Hello World Typescript
```

Comment:
Maybe the language name should be the prefix, e.g. "TypeScript - Hello World" to group language templates together in alphabetical output?

Reply:
fixed.